src/com/stanimiriliev/chaptertwo/heterogeneoustree/products/Egg.java

```diff
@@ -0,0 +1,15 @@
+package com.stanimiriliev.chaptertwo.heterogeneoustree.products;
+
+/**
+ * @author Stanimir Iliev
+ * @since 21.07.2017
+ */
+public class Egg extends Products {
```

Comment:
So if I make these classes (Egg, Milk, Coffee) to implements Comparable interface and not to extend class Products I will be able to compare Egg and Milk for example?